womtool/src/main/scala/womtool/validate/Validate.scala

```diff
@@ -5,15 +5,32 @@ import womtool.WomtoolMain.{SuccessfulTermination, Termination, UnsuccessfulTerm
 import womtool.input.WomGraphMaker
 
 object Validate {
-  def validate(main: Path, inputs: Option[Path]): Termination = if (inputs.isDefined) {
-    WomGraphMaker.fromFiles(main, inputs) match {
-      case Right(_) => SuccessfulTermination("Success!")
-      case Left(errors) => UnsuccessfulTermination(errors.toList.mkString(System.lineSeparator))
+
+  def validate(main: Path, inputs: Option[Path], listDependencies: Boolean): Termination = {
```

Comment:
TOL: `womtool validate` seems like a slightly strange command to add this flag to. Ideally I guess it would be `womtool describe`, if such a command existed. I guess leave it as-is for now, but I might start complaining louder if more "augmentations" get added to this command.

Reply:
Based on the use case mentioned in the ticket, I figured it was supposed to be added to `validate` command. But we can definitely change this to `womtool describe` at a later point.